src/policy/inputs.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_POLICY_INPUTS_H
+#define BITCOIN_POLICY_INPUTS_H
+
+#include <string>
+
+class CCoinsViewCache;
+class CTransaction;
+
+    /**
```

Comment:
I know it's just a move, but maybe you could fix the indentation here and below?